core/src/com/unciv/models/ruleset/Unique.kt

```diff
@@ -81,7 +81,8 @@ object UniqueTriggerActivation {
                     city.population.population += 1
                     city.population.autoAssignPopulation()
                 }
-            "Free Technology" -> civInfo.tech.freeTechs += 1
+            "Free Technology" -> if (!civInfo.isSpectator()) civInfo.tech.freeTechs += 1
+            "[] Free Technologies" -> if (!civInfo.isSpectator()) civInfo.tech.freeTechs += unique.params[0].toInt() 
```

Comment:
Since this adds the "[] Free Technologies" unique, perhaps "Free technology" should be deprecated and replaced by "[1] Free Technologies" to limit duplication.

Reply:
The single/multiple phrasing doesn't really accommodate both in one sentence, this is good